frontend/src/controllers/mano-aloe.service.ts

```diff
@@ -86,6 +86,18 @@ export default class ManoAloeService {
         return this.getCount('gallery');
     }
 
+    public getMultiGallery(): Promise<MultiArtwork[]> {
+        return fetch(this.apiURL + 'multigallery')
+            .then((res: { json: () => any; }) => {
+                return res.json();
+            })
+            .then((apiResponse: MultiGalleryResponse) => {
+                return apiResponse.multigallery.map(multiArtworkFromJson);
```

Comment:
![image](https://user-images.githubusercontent.com/38551592/120918634-313f5800-c66a-11eb-9d85-1ff537795346.png)
huh

Reply:
this happens when the multigallery api returns nothing

Reply:
if multigallery become `undefined` it will cause the above error

Reply:
Is this any different from any of the other dbs that end up empty today? If we're going to address it we probably want to address it for everything, right?

Reply:
It was the backend and the frontend middleware. Backend was sending a `{}` and the frontend expected a MultiArtwork created from a json of `{metadata: {key_value_pairs}, gallery: [str]}`. Since the operation wasn't possible, it returns an undefined value which couldn't be used by the service.